src/app/screens/signBatchPsbtRequest/index.tsx

```diff
@@ -0,0 +1,763 @@
+import { ExternalSatsMethods, MESSAGE_SOURCE } from '@common/types/message-types';
+import AccountHeaderComponent from '@components/accountHeader';
+import BottomModal from '@components/bottomModal';
+import ActionButton from '@components/button';
+import InputOutputComponent from '@components/confirmBtcTransactionComponent/inputOutputComponent';
+import InfoContainer from '@components/infoContainer';
+import RecipientComponent from '@components/recipientComponent';
+import TransactionDetailComponent from '@components/transactionDetailComponent';
+import useDetectOrdinalInSignBatchPsbt from '@hooks/useDetectOrdinalInSignBatchPsbt';
+import useSignBatchPsbtTx from '@hooks/useSignBatchPsbtTx';
+import useWalletSelector from '@hooks/useWalletSelector';
+import { ArrowLeft, ArrowRight, CheckCircle } from '@phosphor-icons/react';
+import { getBtcFiatEquivalent, parsePsbt, satsToBtc } from '@secretkeylabs/xverse-core';
+import { isLedgerAccount } from '@utils/helper';
+import BigNumber from 'bignumber.js';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { NumericFormat } from 'react-number-format';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { MoonLoader } from 'react-spinners';
+import styled from 'styled-components';
+import BundleItemComponent from './bundleItemComponent';
+import BundleItemsComponent from './bundleItemsComponent';
+import CircularProgress from './circularProgress';
+// TODO: Uncomment the commented code in this file when making ledger batch signing work
```

Comment:
we can remove the commented code for now and only add it when we enable the feature for ledger